lib/dor/services/client/object_version.rb

```diff
@@ -79,7 +78,7 @@ def open(**params)
         # @return [String] a message confirming successful closing
         def close(**params)
           resp = connection.post do |req|
-            req.url close_version_path
+            req.url with_querystring(url: close_version_path, params: params)
             req.headers['Content-Type'] = 'application/json'
             req.body = params.to_json if params.any?
```

Comment:
does the `req.body = ...` need to be dropped here like it was above?

Reply:
Yes. Fixed.